Src/CSharpier/DocTypes/Doc.cs

```diff
@@ -2,6 +2,11 @@ namespace CSharpier.DocTypes;
 
 internal abstract class Doc
 {
+    public override string ToString()
+    {
+        return DocSerializer.Serialize(this);
```

Comment:
I'm surprised it took this long for that to happen :grin:

Reply:
So nice for debugging, I don't know why I didn't look into how to do this like 12 months ago.